io.mpv.Mpv.appdata.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2012-2019 mpv.io -->
+<component type="desktop">
```

Comment:
this is just CLI, right? if so this should be console-application, right? https://www.freedesktop.org/software/appstream/docs/sect-Metadata-ConsoleApplication.html

Reply:
GUI mode with drag'n'drop exists.